logitech_r400/src/logitech_r400/logitech_r400

```diff
@@ -64,23 +65,30 @@ class LogitechR400Usb(object):
 
     def __exit__(self, type, value, traceback):
         for device in self._devices:
-            self.release_device(device)
-
-    def is_R400(self, dev):
-        if (dev.idVendor == R400_ID_VENDOR and dev.idProduct == R400_ID_PRODUCT) \
-           or (dev.idVendor == R400_ID_VENDOR_2 and dev.idProduct == R400_ID_PRODUCT_2):
-            return True
+            self._release_device(device)
+
+    def _read_config(self):
+        configs_file_path = rospkg.RosPack().get_path('logitech_r400') + '/config/config.yaml'
+        with open(configs_file_path, 'r') as stream:
+            self.configs = yaml.load(stream)
+
+    def _is_R400(self, dev):
+        for config_name, config in self.configs.iteritems():
+            if (dev.idVendor == int('0x' + str(config['vendor']), 16) and
```

Comment:
You might do this more easily by defining the yaml parameter as a hexadecimal number:

```yaml
  product: 0x0e00
  vendor: 0x0111
```

Then this is simply:

```python
if (dev.idVendor == config['vendor']) and
```